packages/skill-evolution/hooks/capture-skill-telemetry.py

```
@@ -0,0 +1,130 @@
+            print(f"Hook error (capture-skill-telemetry): {e}", file=sys.stderr)
+            sys.exit(0)
+
+    return wrapper
+
+
+def _project_dir(data: dict) -> str:
+    return data.get("cwd") or os.environ.get("CLAUDE_PROJECT_DIR", os.getcwd())
+
+
+def _response_text(data: dict) -> str:
+    """tool_response を文字列化する（dict/list でも探索）。"""
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)